static/js/apps/homepage/app.tsx

```diff
@@ -56,24 +59,41 @@ export function App({
   sampleQuestions,
   routes,
 }: AppProps): ReactElement {
-  const topicLinkChips: LinkChip[] = topics.map((topic) => ({
+  const topicLinkChips: Link[] = topics.map((topic) => ({
     id: topic.id,
     title: topic.title,
     url: topic.browseUrl,
   }));
 
   return (
-    <>
-      <HeroVideo routes={routes} />
-      <LinkChips
-        title={"Topics to explore"}
-        section="topic"
-        linkChips={topicLinkChips}
-      />
-      <SampleQuestions sampleQuestions={sampleQuestions} />
-      <Tools routes={routes} />
-      <Build routes={routes} />
-      <Partners partners={partners} gaEvent={GA_EVENT_HOMEPAGE_CLICK} />
-    </>
+    <ThemeProvider theme={theme}>
+      <Section colorVariant="light">
+        <HomeHero linkChips={topicLinkChips} />
+      </Section>
+
+      <Section>
+        <SampleQuestions sampleQuestions={sampleQuestions} />
+      </Section>
+
+      <Section variant="small">
+        <Tools routes={routes} />
+      </Section>
+
+      <Separator border={false} variant="small" />
+
+      <Section variant="large" colorVariant="dark">
+        <BuildYourOwn routes={routes} />
```

Comment:
Looks like the figma has some copy updates for this page:

<img width="1720" alt="Screenshot 2024-12-06 at 10 43 57 PM" src="https://github.com/user-attachments/assets/d67cbb26-18f5-4ed7-9529-d23539811034">